WebApi/WebApi/Controllers/CommentController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using WebApi.Models.DTO;
+using WebApi.Services.Services_Interfaces;
+
+namespace WebApi.Controllers
+{
+    [ApiController]
+    [Route("api/comment")]
+    public class CommentController
+    {
+        private readonly ICommentService _commentService;
+        private readonly ILogger<CommentController> _logger;
+        public CommentController(ILogger<CommentController> logger, ICommentService commentService)
+        {
+            _commentService = commentService;
+            _logger = logger;
+        }
+        [HttpGet]
+        public IQueryable<CommentDTO> GetAll([FromHeader] int userId)
+        {
+            return _commentService.GetAll(userId);
```

Comment:
Powinien być utworzony nowy obiekt DTO do zwracania, pola zgodne z dokumentacją (można na razie zakomentować zasoby, które jeszcze nie powstały)
Należy wyliczyć ownerMode (tzn. porównać userID z Posts z userID z headera) - zrobić to w CommentsService
Nowe DTO powinno wyglądać jak tu:
"comments": [
                  {
                    "id": "155",
                    "ownerMode": "true",
                    "content": "Post Content",
                    "date": "20-15-2020",
                    "author": {
                      "name": "Oliver",
                      "surname": "Wring"
                    },
                    "isLikedByUser": "true",
                    "likesCount": "10"
                  }
                ],